user-guide/Advanced_Functionality/Databases/STaaS/STaaS.md

```diff
@@ -21,13 +16,96 @@ Advantages of DataMiner Storage as a Service (STaaS) include:
 
 - Outstanding performance, scalability, and redundancy
 
-- Compatibility with all data types
-
 - Cost-effectiveness
 
 - Safe and secure data
 
 > [!TIP]
->
-> - To read more about Storage as a Service and its advantages, see [DataMiner STaaS: a game-changer for your storage needs](https://community.dataminer.services/dataminer-staas-a-game-changer-for-your-storage-needs/).
-> - As Storage as a Service is **scheduled for production in Q3 of 2023**, we are working on adding more information to this section. Check back for updates frequently!
+> If you have any questions or need support with the STaaS feature, contact <staas@dataminer.services>.
+
+## Setting up STaaS
+
+1. [Upgrade your DataMiner System](xref:Upgrading_a_DataMiner_Agent) to version 10.3.10 [CU1] or higher.
+
+   > [!IMPORTANT]
+   > We recommend always upgrading DataMiner to the latest available version to get the latest features and performance updates.
+
+1. Make sure your DataMiner System is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
+
+1. Retrieve your organization and coordination IDs:
```

Comment:
What do you need these for? This isn't mentioned elsewhere in the procedure.

Reply:
since the registration should be handled by the TAM/Skyline representative, this can be remove